modules/administration_manual/pages/installation/manual_installation_ubuntu_16_04.adoc

```diff
@@ -0,0 +1,163 @@
+= Manual Installation on Linux - Ubuntu 16.04
+:nginx-app-tracing-url: https://www.nginx.com/blog/application-tracing-nginx-plus/
+
+== Introduction
+This guide describes how to install the required dependencies, the database and ownCloud itself in a
+single server setup.
+
+
+[[install-the-required-packages]]
```

Comment:
You don't need to create custom anchors, as anchors are [auto-generated for headers](https://asciidoctor.org/docs/user-manual/#auto-generated-ids).